README.md

```diff
@@ -697,6 +697,7 @@ A curated list of awesome Java frameworks, libraries and software.
 * [Java, SQL, and jOOQ](http://blog.jooq.org/)
 * [Java.net](http://java.net/)
 * [Javalobby](http://java.dzone.com/)
+* [JavaTutorial](http://docs.oracle.com/javase/tutorial/)
```

Comment:
Java SE is the core runtime library, and JavaTutorials is the oldest, official, complete, and free resource for beginners to learn Java. I believe this should be added to list in favor of beginners who wish to start learning Java.